turbostack_configuration/Database/phpMyAdmin.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Er is ook een linkje nu beschikbaar onder accounts , database info (en ook dat gaat nog uitgebreid worden)

![image](https://github.com/user-attachments/assets/ca181075-0757-4bdc-854d-4b915de21f5e)